server/submissionWorkflow/queries.ts

```diff
@@ -0,0 +1,42 @@
+import { Collection, SubmissionWorkflow } from 'server/models';
+// import { SubmissionWorkflow as SubmissionWorkflowType } from 'types';
+
+export const createSubmissionWorkflow = async ({
+	collectionId,
+	enabled,
+	instructions,
+	afterSubmittedText,
+	email,
+	layoutBlock,
+}) => {
+	return Collection.findOne({ where: { id: collectionId } }).then(() =>
```

Comment:
Good thinking here, but be aware that this promise will resolve whether or not a Collection was actually found.